hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServer.java

```diff
@@ -64,10 +64,11 @@ public class NettyRpcServer extends RpcServer {
   /**
    * Name of property to change netty rpc server eventloop thread count. Default is 0. Tests may set
    * this down from unlimited.
+   * @deprecated Use NettyEventLoopGroupConfig#NETTY_WORKER_COUNT_KEY instead.
    */
+  @Deprecated
```

Comment:
Does IA.LP means we will have to go through deprecation cycle?

Reply:
In my opinion we can deprecate it in 2.x and remove it in the 3.0.0 final release, and that is being pedantic, because it is unlikely to have been used. WDYT @virajjasani @Apache9 ?

Reply:
If you agree, I would apply this PR to the branch-2s and just remove the constant in master branch.

By the way the class is marked LP(Config) because the class itself -- NettyRpcServer -- is a configuration option, for rpc.server.impl.

Reply:
Sure, it makes sense removing from 3.0 and keep/mark it deprecated in 2.x.